src/utils.c

```diff
@@ -345,6 +335,16 @@ void r_int_fill_seq(SEXP x, int start) {
   }
 }
 
+SEXP r_seq(R_len_t from, R_len_t to) {
+  R_len_t n = to - from;
```

Comment:
This definitely needs a check

Reply:
`Rf_allocVector()` checks for negative inputs, but I added another check.